editor/src/clj/editor/tile_map.clj

```diff
@@ -1141,7 +1141,7 @@
   (output input-handler Runnable :cached (g/constantly (make-input-handler)))
   (output info-text g/Str (g/fnk [cursor-world-pos tile-dimensions]
                             (when-some [[x y] (get-current-tile cursor-world-pos tile-dimensions)]
-                              (format "Cell: %d, %d" (+ 1 x) (+ 1 y))))))
+                              (format "Cell: %d, %d" (/ x :tile-width) (/ y :tile-height))))))
```

Comment:
This is very wrong. You're dividing an integer by a keyword. This will result in an exception at runtime.

It also won't produce the result you're after. The tile ID is a single number that should be appended to the existing string, not replacing the existing tile coordinates. The tile ID should be the index of the tile among all the tiles in the tile map, which you should be able to calculate from x + y * tiles-per-row, but you need to figure out the tiles-per-row somehow.